### ISSUE TITLE
StopCommand issue on Windows

### ISSUE THREAD

**Issue by yasargil** [eclipse-vertx/vert.x#1449]
On Windows the stop command can match multiple process when the vertx.id is similar.

e.g. In case of the two running processes: 'app' and 'app-2'.
the command 'stop app' will stop both of them.

**Comment by cescoffier** [comment]
Hi,

Yes. Do you now the command to match exactly one ?

**Comment by yasargil** [comment]
No. I don't think there is a solution with the like command because you need to search a substring. 
One workaround would be to internally append a suffix to the vertex-id and cut it of in the list command. 
e.g "app" becomes "app-(VERTXIDEND)" and "app-2" becomes "app-2-(VERTXIDEND)"
